.github/workflows/build-m1-binaries.yml

```diff
@@ -26,13 +26,15 @@ jobs:
           echo $PATH
           . ~/miniconda3/etc/profile.d/conda.sh
           set -ex
-          export BUILD_VERSION=0.14.0.dev$(date "+%Y%m%d")
+          . packaging/pkg_helpers.bash
+          setup_build_version
           WHL_NAME=torchaudio-${BUILD_VERSION}-cp${PY_VERS/.}-cp${PY_VERS/.}-macosx_11_0_arm64.whl
-          conda create -yp ${ENV_NAME} python=${PY_VERS} numpy cmake ninja libpng openjpeg wheel pkg-config
-          conda run -p ${ENV_NAME} python3 -mpip install torch --extra-index-url=https://download.pytorch.org/whl/nightly
+          conda create -yp ${ENV_NAME} python=${PY_VERS} numpy cmake pytest ninja libpng openjpeg wheel pkg-config
```

Comment:
nit: seems like the unit test step was removed in the end. `pytest` is not necessary in that case.
(also `libpng` and `openjpeg` have nothing to do in audio repo)